doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -1889,18 +1889,35 @@ Parentheses are allowed, and ``[[AS] correlation-name]`` is allowed.
 
 The maximum number of joins in a FROM clause is 64.
 
+The ``SEQSCAN`` keyword (since :doc:`2.11 </release/2.11.0>`) marks the queries that
+perform sequential scans during the execution. It happens if the query can't use indexes,
+and goes through all the table rows one by one, sometimes causing a heavy load.
+Such queries are called *scan queries*. If a scan query doesn't have the
+``SEQSCAN`` keyword, Tarantool raises an error. ``SEQSCAN`` must precede all
+names of the tables that the query scans.
+
+To find out if a query performs a sequential scan, use ``EXPLAIN QUERY PLAN``.
+For scan queries, the result contains ``SCAN_TABLE tabe_name``.
```

Comment:
It is actually `SCAN TABLE table_name` instead of `SCAN_TABLE tabe_name`. See example in the comment above.